packages/v4/patternfly-docs/content/design-guidelines/components/accordion/accordion.md

```diff
@@ -3,39 +3,35 @@ id: Accordion
 section: components
 ---
 
-An **accordion** is an interactive container that expands and collapses to hide or reveal the nested content. It takes advantage of progressive disclosure to help reduce page scrolling by allowing users to choose to show or hide more detailed information as needed.
+An **Accordion** is an interactive container that expands and collapses to hide or reveal nested content. It takes advantage of progressive disclosure to help reduce page scrolling, by allowing users to choose whether they want to show or hide more detailed information as needed.
```

Comment:
Are we capitalizing the component names now? Just want to know whether that's something I need to look out for in these reviews.